we change some variables in the Makefile
for a better handling in gentoo ebuild

Signed-off-by: Joerg Bornkessel <hd_brummy@gentoo.org> ( 22 Dec 2019 )
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@
 
 # Output control
 
-ifdef VERBOSE
+ifeq ($(VERBOSE),1)
 Q =
 else
 Q = @
@@ -89,7 +89,7 @@
 ifdef HDRDIR
 HDRDIR   := -I$(HDRDIR)
 endif
-ifndef NO_KBD
+ifeq ($(USE_KBD),1)
 DEFINES += -DREMOTE_KBD
 endif
 ifdef REMOTE
@@ -98,12 +98,12 @@
 ifdef VDR_USER
 DEFINES += -DVDR_USER=\"$(VDR_USER)\"
 endif
-ifdef BIDI
+ifeq ($(BIDI),1)
 INCLUDES += $(shell pkg-config --cflags fribidi)
 DEFINES += -DBIDI
 LIBS += $(shell pkg-config --libs fribidi)
 endif
-ifdef SDNOTIFY
+ifeq ($(SDNOTIFY),1)
 INCLUDES += $(shell pkg-config --silence-errors --cflags libsystemd-daemon || pkg-config --cflags libsystemd)
 DEFINES += -DSDNOTIFY
 LIBS += $(shell pkg-config --silence-errors --libs libsystemd-daemon || pkg-config --libs libsystemd)
